stl/inc/mdspan

```diff
@@ -397,12 +395,12 @@ public:
     constexpr mapping() noexcept               = default;
     constexpr mapping(const mapping&) noexcept = default;
 
-    constexpr mapping(const _Extents& e) noexcept : _Myext(e){};
+    constexpr mapping(const _Extents& _Ext) noexcept : _Myext(_Ext) {}

```

Comment:
FWIW `e` is reserved (probably?) since C++20 by `std::numbers::e`, and the identifier `result` changed in the previous PR is also reserved since C++20 by `std::chrono::local_info::result`. No change requested (since `_Ext` is a bit clearer).

Reply:
We conventionally never take advantage of the Standard indirectly forbidding macroization of pretty identifiers, except for `type` and `value` in internal type traits (not even as local variables).